test/clj_sqs_extended/core_test.clj

```diff
@@ -0,0 +1,28 @@
+(ns clj-sqs-extended.core-test
+  "Basic tests for the primary API of `clj-extended-sqs`."
+  (:require [clojure.test :refer [deftest is testing use-fixtures]]
+            [clj-sqs-extended.core :as sqs-ext]
+            [clj-sqs-extended.test-fixtures :refer [with-localstack-environment
+                                                    localstack-sqs
+                                                    test-queue]]))
+
+
+(use-fixtures :once with-localstack-environment)
+
+(def ^:private test-message "What hath God wrought?")
+
+
+(deftest ^:unit test-send-receive-message
+  (testing "Sending/Receiving message"
+    (let [url (.getQueueUrl (test-queue))]
+      (sqs-ext/send-message-on-queue (localstack-sqs) url test-message)
+      (let [reply (sqs-ext/receive-messages-on-queue (localstack-sqs) url)]
+        (is (= 1 (count reply)))
+        (is (= test-message (.getBody (first reply))))))))
```

Comment:
The Java object further includes the message id (maybe not very relevant) but it also includes the receiptHandle which will be used to delete specific messages from an SQS queue. For this I see a usecase, so we could maybe return a map with the 3 fields instead of the Java object?

Reply:
@daisybytes sounds good. we can handle this within this library's implementation but not expose them to the end user in the final `handle-queue`.